include/deal.II/multigrid/mg_transfer_global_coarsening.h

```diff
@@ -261,7 +261,7 @@ class MGTwoLevelTransferBase : public Subscriptor
    * Enable inplace vector operations if external and internal vectors
    * are compatible.
    */
-  virtual void
+  virtual std::pair<bool, bool>
```

Comment:
Could you please add a short commend explaining the return argument (success in using the coarse/fine partitioner handed in)? Here and for the same interfaces further down.